pylsp_black/_utils.py

```diff
@@ -0,0 +1,11 @@
+# Eol chars accepted by the LSP protocol
+EOL_CHARS = ["\r\n", "\r", "\n"]
+
+
+def get_eol_chars(text):
```

Comment:
I also need to add this function in `python-lsp-server` to check for eols there.

Should we import it from there or do you prefer to leave it here?

Reply:
See python-lsp/python-lsp-server#151

Reply:
@krassowski already did a lot of benchmarks and the regex one is way more efficient. So, I'll use his version and import it from `python-lsp-server` here.

Reply:
Great, thanks for the update.